templates/pages/checkout.html

```diff
@@ -1 +1,34 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
+    {{{ open_sans_stylesheet }}}
+    {{{ ng_checkout_stylesheet }}}
+    {{{ stylesheet '/assets/css/optimized-checkout.css' }}}
+    {{ OptimizedCheckoutWebdavCss }}
```

Comment:
@snaderiBC: the `getFontsCollection` actually is a handlebar helper within the **paper** repo https://github.com/bigcommerce/paper/blob/master/helpers/getFontsCollection.js#L8 which is a plugin for `stapler`